src/kernels/kernel.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import type * as nbformat from '@jupyterlab/nbformat';
+import * as nbformat from '@jupyterlab/nbformat';
```

Comment:
Any reason we are not just `import type`anymore?

Reply:
thats weird, don't remember chaning that.